lib/beaker-hostgenerator/data.rb

```diff
@@ -191,6 +191,19 @@ def osinfo
             'template' => 'cisco-nxos-9k-x86_64'
           }
         },
+        'cisconxhw-64' => {
+          :general => {
+            'platform'           => 'cisco_nexus-7-x86_64',
+            'packaging_platform' => 'cisco-wrlinux-5-x86_64',
+            'vrf' => 'management',
+            'ssh' => {
+              'user' => 'devopsuser'
+            }
+          },
+          :abs => {
+            'template' => 'cisco-nxos_hw-9k-x86_64'
```

Comment:
should we have a test fixture for that?

Reply:
I used the rake task to generate the test fixtures that are in the PR now, should I make a new fixture to test this?